tools/mitre/const.py

```diff
@@ -76,11 +82,15 @@
 IMPACT_TYPE_j= 'x_mitre_impact_type'
 PERMISSIONS_REQ_j= 'x_mitre_permissions_required'
 SYSTEM_REQ_j= 'x_mitre_system_requirements'
+PHASES_j= 'kill_chain_phases'
 
 # Relationship type
 RELATIONSHIP_j= 'relationship'
 RELATIONSHIP_TYPE_j= 'relationship_type'
 REVOKED_BY_j= 'revoked-by'
 SOURCE_REF_j='source_ref'
 TARGET_REF_j= 'target_ref'
+DEPRECATED_j='x_mitre_deprecated'
+SHORT_NAME_j='x_mitre_shortname'
```

Comment:
Short name and phase name are not relationship types. Move them where you located kill chain phases.